gym_sts/spaces/observations/__init__.py

```diff
@@ -0,0 +1 @@
+from .observations import OBSERVATION_SPACE, Observation  # noqa: F401
```

Comment:
Why are we exporting these two?

Reply:
They were previously exported from `gym_sts.spaces.observations`, so this is just for parity, rather than updating the import in `gym_sts.envs.base`. I just try to avoid overly verbose imports if they can be abstracted (in this case, avoiding `gym_sts.spaces.observations.observation`).

Reply:
Sure, I buy that